administrator/components/com_admin/postinstall/htaccessbrotli.php

```diff
@@ -26,5 +26,11 @@
  */
 function admin_postinstall_htaccessbrotli_condition()
 {
-    return true;
+    $htaccessContent = '';
+
+    if (is_file(JPATH_ROOT . '/.htaccess') || is_file(JPATH_ROOT . '/htaccess.txt')) {
+        $htaccessContent = file_get_contents(is_file(JPATH_ROOT . '/.htaccess') ? JPATH_ROOT . '/.htaccess' : JPATH_ROOT . '/htaccess.txt');
+    }
```

Comment:
```suggestion
    $htaccessContent = is_file(JPATH_ROOT . '/.htaccess') ? file_get_contents(is_file(JPATH_ROOT . '/.htaccess') : '';
```
So tät' ich das machen tun :-)

Reply:
Nee, nicht gut.